### ISSUE TITLE
Rename Respositories

### ISSUE THREAD

**Issue by jmcc0nn3ll** [jetty/.eclipsefdn#28]
When we decomposed the toolchain, we kept the original naming conventions of the EF regarding our repository names.  

We are kicking around a mass rename to essentially follow maven naming conventions.

jetty.websocket.api - > jetty-websocket-api
jetty.servlet.api -> jetty-servlet-api
jetty.setuid -> jetty-setuid
jetty.test.helper -> jetty-test-helper
jetty.perf.helper -> jetty-perf-helper
jetty.alpn.api - > jetty-alpn-api
jetty.artifact.resources -> jetty-artifact-remote-resources
jetty.assembly.descriptors -> jetty-assembly-descriptors
jetty.schemas -> jetty-schemas
jetty.schemas.xhtml -> jetty-xhtml-schemas
jetty.docker -> jetty-docker
jetty.parent -> jetty-parent
jetty.test.policy -> jetty-test-policy

We are also thinking of fixing jetty.project so:

jetty.project -> jetty

The website is in a bit of flux but eventually:
jetty.website.2020 -> jetty-website-2020
jetty-website -> jetty-website-2023 
jetty.website -> jetty-website ( or just website ) once jetty.org is deployable

**Comment by jmcc0nn3ll** [comment]
@netomi This is only a discussion right now, please don't make any changes, but I wanted to ask if repo rename support is in otterdog right now or if that is something we would have to flag you specifically for.

Sorry to be a bother, but we are hoping to get things squared away naming-wise before we start sorting out the #18.

**Comment by netomi** [comment]
it is supported.

In order to rename a repo, just give it a new name, and add a aliases: ["oldname"] property for the service to be able to associate it.

Example PR can be found here: https://github.com/eclipse-uprotocol/.eclipsefdn/pull/17

**Comment by jmcc0nn3ll** [comment]
I was able to rename the jetty.test.policy repository as expected, you just can't rename an archived repo.  

Time to discuss the issue amongst the committers and decide.

**Comment by netomi** [comment]
well we can rename archived repos, it just requires an additional manual step. I would resort on doing things like that, if they are necessary we can ofc.

**Comment by jmcc0nn3ll** [comment]
@netomi yes, sorry, I put that there as a bit of a reminder that we can't do it directly
